src/pages/Profile.tsx

```diff
@@ -0,0 +1,31 @@
+import { Box } from "../utilities/material-ui/material-components";
+import {
+  ProfileSection,
+  TrendingSection,
+  AddPostMobile,
+  Sidenav,
+} from "../components/index";
+import React from "react";
+
+export function Profile() {
+  const [open, setOpen] = React.useState(false);
+  const handleOpen = () => setOpen(true);
+
+  return (
+    <Box
+      sx={{
+        display: "flex",
+        p: { md: 0, xs: 2 },
+        gap: 2,
+        pr: { md: 2 },
```

Comment:
seems redundant to have both overall padding and padding right for medium sized screens

Reply:
I'll fix it in the next PR.